### PR TITLE
packaging: fix the broken usage help

### PR DESCRIPTION
**Pull request by deagon**
Fix typos in usage and move the logic of valid check after usage help print.

Fixes: #9386

### CODE REVIEW

File: tools/packaging/guest-image/build_se_image.sh [hunk 1]
```diff
@@ -18,9 +18,6 @@ source "$kata_root_dir/ci/lib.sh"
 source "${packaging_root_dir}/scripts/lib.sh"
 
 ARCH=${ARCH:-$(uname -m)}
-if [ $(uname -m) == "${ARCH}" ]; then
-	[ "${ARCH}" == "s390x" ] || die "Building a Secure Execution image is currently only supported on s390x."
-fi
```
Comment (beraldoleal) [inline tools/packaging/guest-image/build_se_image.sh:23]: Your commit message says is fixing only typos... so maybe this is part of another commit?

Do you mind explaining why did you moved the section?

**Review by beraldoleal — CHANGES_REQUESTED** [review]

File: tools/packaging/guest-image/build_se_image.sh (hunk 1 above)
Comment (deagon) [reply tools/packaging/guest-image/build_se_image.sh:23]: If I don't move this section after usage function call, the output is still wrong as follows.

```
# tools/packaging/guest-image/build_se_image.sh -h
ERROR: Building a Secure Execution image is currently only supported on s390x.
```

In other words, the logic of valid check might should after usage print.

My original intention of the PR is `fix the broken usage help` as the subject shows.

**Review by beraldoleal — COMMENTED** [review]
Left a minor comment above, to improve the commit message. But is nit.

File: tools/packaging/guest-image/build_se_image.sh (hunk 1 above)
Comment (deagon) [reply tools/packaging/guest-image/build_se_image.sh:23]: It's ok for me. Thanks a lot for your review.

**Review by beraldoleal — APPROVED** [review]
lgtm, thanks

**Comment by deagon** [comment]
@justxuewei @Apokleos Take a review please.

**Review by GabyCT — APPROVED** [review]

**Comment by beraldoleal** [comment]
Hi @deagon, sorry for the long hiatus here, our bad. :(

If you are still interested, please rebase and I will merge it if passing all tests.

**Comment by deagon** [comment]
> Hi @deagon, sorry for the long hiatus here, our bad. :(
> 
> If you are still interested, please rebase and I will merge it if passing all tests.

Sure, sorry for my late.

Using slash will be better for human-reading, maybe.

The new commit would not change other places.

**Comment by deagon** [comment]
@beraldoleal Take a review again, please.

**Comment by beraldoleal** [comment]
lgtm, thanks.

**Comment by deagon** [comment]
> lgtm, thanks.

@beraldoleal  It seams to need a /ok-to-test label for triggers.

**Review by justxuewei — APPROVED** [review]
Lgtm, thanks! Sorry for the late reply too. I have triggered the tests. Let us get this merged after the tests finish.

**Comment by justxuewei** [comment]
Umm, It's so weird. Looks like "WIP Check" doesn't work as expectedly, and can't check its details. Would you mind rebasing onto the master again? @deagon

**Comment by justxuewei** [comment]
Hi @deagon - Please ping me if you encounter any issues. After forced push I can't see any information about the CI issues. Thanks ;)

**Comment by deagon** [comment]
> Hi @deagon - Please ping me if you encounter any issues. After forced push I can't see any information about the CI issues. Thanks ;)

@justxuewei Just rebase it to test if passed by the gatekeeper. No more issues.

**Comment by justxuewei** [comment]
You can ignore items that are not marked as "Required", e.g. "Gatekeeper" in this case.

**Comment by deagon** [comment]
@justxuewei Got it!

**Pull request merged**